foundationdb/datadog_checks/foundationdb/check.py

```diff
@@ -0,0 +1,219 @@
+from datadog_checks.base import AgentCheck
+import json
+from typing import Any
+import fdb
+
+fdb.api_version(600)
```

Comment:
WDYTA making this configurable within `init_config`?

Reply:
Quoting the FoundationDB docs:

> The purpose of this design is to allow the server, client libraries, or bindings to be upgraded without having to modify application code. 

So no, this shouldn't be configurable, because it's about the API that we're using to implement the integration, and we'd have to review and maybe change the Python code here in the event we did want to increment this in the future.